SDK/AppCenterData/Microsoft.AppCenter.Data.iOS.Bindings/ApiDefinition.cs

```diff
@@ -201,6 +223,19 @@ interface MSWriteOptions : MSBaseOptions
     {
     }
 
+    // @protocol MSRemoteOperationDelegate <NSObject>
+    [Protocol, Model]
+    [BaseType(typeof(NSObject))]
+    interface MSRemoteOperationDelegate
+    {
+        // @optional - (void)data:(MSData *)data
+        //               didCompleteRemoteOperation:(NSString*) operation
+        //                      forDocumentMetadata:(MSDocumentMetadata* _Nullable) documentMetadata
+        //                                withError:(MSDataError* _Nullable) error;
+        [Export("data:didCompleteRemoteOperation:forDocumentMetadata:withError:")]
+        void DataDidCompletedRemoteOperation(MSData data, string operation, MSDocumentMetadata documentMetadata, MSDataError error);
```

Comment:
need to rename to DataDidCompleteRemoteOperation

Reply:
marked as resolved, but no new commit

Reply:
pushed now